layouts/shortcodes/config-uri-gcp-basic-auth.html

```diff
@@ -1 +1 @@
-https://raw.githubusercontent.com/kubeflow/kubeflow/cfb336b7/bootstrap/config/kfctl_gcp_basic_auth.0.6.2.yaml
\ No newline at end of file
+https://raw.githubusercontent.com/kubeflow/manifests/v0.7-branch/kfdef/kfctl_gcp_basic_auth.yaml
```

Comment:
This is now kfctl_gcp_basic_auth.0.7.0.yaml

Reply:
Done.